src/index.html

```diff
@@ -2,422 +2,480 @@
+                                <h5 class="card-title">Student</h5>
+                                <div>
+                                    <button class="btn btn-sm info-btn text-light" disabled id="join-club">
+                                        Applications are closed
+                                    </button>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <img alt="Volunteers planning club logistics" class="card-img-top"
+                                 src="/assets/img/volunteer-background.jpg">
+                            <div class="card-body">
+                                <h5 class="card-title">Volunteer</h5>
+                                <div>
+                                    <a class="btn btn-sm info-btn text-light"
+                                       href="/assets/pdf/GCL-Volunteer-Information.pdf"
+                                       id="volunteerInfo"
+                                       rel="noopener" target="_blank">
+                                        Information
+                                    </a>
+                                    <a class="btn btn-sm cta-btn" href="https://www.surveymonkey.com/r/2JNS682"
+                                       id="joinTeam"
+                                       rel="noopener" target="_blank">
+                                        Join our team
+                                    </a>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
                 </div>
-              </div>
             </div>
-          </div>
-        </div>
-      </section>
+        </section>
 
-      <section id="image-gallery" class="w-100">
-        <div id="image-carousel" class="carousel slide carousel-fade h-100" data-ride="carousel">
-          <ol class="carousel-indicators">
-            <!-- individual indicators programatically appended in index.js -->
-          </ol>
-          <div class="carousel-inner">
-            <!-- individual items of gallery programatically appended in index.js -->
-          </div>
-        </div>
-      </section>
+        <section id="podcast" class="col-12 justify-content-center py-5 bg-grey">
+                <div class="text-center col-lg-7 col-xs-8 mx-auto mb-3">
+                    <h2 class="font-weight-bold">Check out our <a href="https://www.buzzsprout.com/1882218"
+                                                                  target="_blank" rel="noopener noreferrer">Podcast</a>!</h2>
+                    <i>
+                        Hosts Elsbeth Magilton and Aakriti Agrawal profile a historical woman in coding each month and
+                        introduce listeners to a coder in or from Nebraska.
+                    </i>
+                </div>
+                <iframe id=player_iframe
```

Comment:
I'm on an ultrawide display and this shows up as: 
![image](https://user-images.githubusercontent.com/22859824/141164424-4a92bbb8-0d73-4a0b-85fd-dd02ff3240e1.png)

Maybe would be worth to wrap this iframe in another div with fewer columns. Maybe could even move it into the above div where you have the header and caption